### ISSUE TITLE
Why header keys are downcased?

### ISSUE THREAD

**Issue by xliiauo** [brooklynDev/airborne#143]
Hi, I was trying to test out the response headers. And the response header is something like `"X-TV-subscriptionId": "12345678"`, and I tried to `expect_header "X-TV-subscriptionId", "12345678"` and it returns false, it will return true only if I do `expect_header "x_tv_subscriptionid", "12345678"`.

Is this behaviour intended and what if I want to test out the exact header key instead of the downcased one (don't know why - is also downcased to _)?

Many thanks in advance.

**Comment by rdalverny** [comment]
Yes, this is expected (downcase, underscore instead of hyphen). This is tied to rest-client implementation:
 * see https://github.com/rest-client/rest-client/blob/aea752218aab3349d0894f58a3854a03057e83cc/lib/restclient/abstract_response.rb#L158-L192
 * and https://github.com/rest-client/rest-client/blob/6beead019b9971f9576e78f78e140a377b3d6d3d/lib/restclient/request.rb#L553-L575

Plus, airborne uses an indifferent access hash, so you may use a string or a symbol as a key.
